content/en/synthetics/troubleshooting/_index.md

```diff
@@ -20,6 +20,10 @@ If you experience issues setting up or configuring Datadog Synthetic Monitoring,
 
 ### Recording
 
+#### My website is not loading in the iframe
+
+After downloading the [Datadog extension][2], you are unable to see your website in the iframe on the right side of your Browser test's recorder and the iframe displays `Your website does not support being loaded through an iframe.`. This could indeed mean that your application has some settings preventing it from being opened in an iframe. If that is the case, try opening your website in a pop up clicking **Open in Popup** to record your journey.
```

Comment:
```suggestion
After downloading the [Datadog extension][2], you are unable to see your website in the iframe on the right side of your Browser test's recorder and the iframe displays `Your website does not support being loaded through an iframe.`. This could mean that your application has some settings preventing it from being opened in an iframe. If that is the case, try opening your website in a pop up by clicking **Open in Popup** to record your journey.
```